doc/system_health_monitoring/system-health-HLD.md

```diff
@@ -275,6 +278,7 @@ When something is wrong
         orchagent is not running
 	Hardware           Fault
         PSU 1 temp 85C and threshold is 70C
+        PSU 1 power (66.32w) exceeds thresholds (warning: 60.00w critical: 70.00w)
```

Comment:
it's hard to say which threshold is crossed because most times it exceeds the critical threshold but sometimes it can be in (warning_suppress_threshold, critical_threshold) because of hysteresis.

Reply:
Updated